CHANGELOG.md

```diff
@@ -1,3 +1,30 @@
+# dbt_jira v1.1.0
+
+## Schema/Data Change
+**7 total changes • 5 possible breaking changes**
+
+| Data Model(s) | Change type | Old | New | Notes |
+| ---------- | ----------- | -------- | -------- | ----- |
+| [jira__timestamp_issue_field_history](https://fivetran.github.io/dbt_jira/#!/model/model.jira.jira__timestamp_issue_field_history)       | New Model (**Breaking Change**)  |     |  | SCD Type 2 table tracking field changes at timestamp level with validity periods. Each record shows complete field state during a time period with `valid_from`/`valid_until` timestamps. |
+| [jira__issue_status_transitions](https://fivetran.github.io/dbt_jira/#!/model/model.jira.jira__issue_status_transitions) | New Model (**Breaking Change**) |  |  | Issue status transition tracking with workflow analysis. Provides chronological view of status changes with timing metrics, transition direction analysis, and lifecycle indicators.  |
+| [stg_jira__team](https://fivetran.github.io/dbt_jira/#!/model/model.jira.stg_jira__team) | New Model (**Breaking Change**) |  |  | Added staging model for Jira teams functionality. You can leverage the `jira__using_teams` variable to disable this model in dbt Core (Quickstart will handle enabling/disabling automatically).  |
+| [stg_jira__team_tmp](https://fivetran.github.io/dbt_jira/#!/model/model.jira.stg_jira__team) | New Model (**Breaking Change**) |  |  | Added temp model for Jira teams functionality.  You can leverage the `jira__using_teams` variable to disable this model in dbt Core (Quickstart will handle enabling/disabling automatically).  |
+| [jira__daily_issue_field_history](https://fivetran.github.io/dbt_jira/#!/model/model.jira.jira__daily_issue_field_history)       | Data Change (**Breaking Change**)  | `project`, `assignee`, `team` brought in with default field id values. | `project`, `assignee`, `team` names now brought through. | Fields customers can define with the `issue_field_history_columns` variable that bring in their identifier values now bring in the names for easier analytical insights. |
```

Comment:
Small adjustment to make sure it doesn't seem that this is applicable to all users. Only to those who have those issue fields included in the `issue_field_history_columns` variable.
```suggestion
| [jira__daily_issue_field_history](https://fivetran.github.io/dbt_jira/#!/model/model.jira.jira__daily_issue_field_history)       | Data Change (**Breaking Change**)  | If configured - `project`, `assignee`, `team` brought in with default field id values. | If configured - `project`, `assignee`, `team` names now brought through. | Fields customers can define with the `issue_field_history_columns` variable that bring in their identifier values now bring in the names for easier analytical insights. |
```